webapp/src/main/webapp/errorhandler.jsp

```diff
@@ -1,54 +1,53 @@
-<%@ page isErrorPage="true" import="java.io.*" %>
+<%@ page isErrorPage="true" import="java.io.*"%>
```

Comment:
It's actually good practice and cleaner to leave this space at the end, therefore it was intentional and I think you should undo this